### PR TITLE
etcd: os faq

### PR DESCRIPTION
**Pull request by ElijahCaine**
There's a few mis-conceptions about etcd. This document clarifies some of that confusion.

Future work:
- Link to this document prominently in:
  - etcd's landing page
  - etcd's docs landing page
  - etcd's getting started page

### CODE REVIEW

File: etcd/os-faq.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# etcd on Container Linux FAQ
+
+Are you planning on running etcd on CoreOS Container Linux? The following FAQ clarifies some common mis-conceptions.
```
Comment (radhikapc) [inline etcd/os-faq.md]: misconceptions. no hyphen.

File: etcd/os-faq.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Why is etcd v2 still in the os?
+
+The etcd binary will be included in the OS until June, 2018.
+
+The preferred way to run etcd in Container Linux is to use the etcd-member service mentioned above in the first question.
+
+## When will v3 be included in the OS image?
+
+Never.
```
Comment (robszumski) [inline etcd/os-faq.md]: this is a little harsh

Comment (joshix) [inline etcd/os-faq.md]: Thank you. I'm glad you noticed. :D
We'll give this a pass for external palatability before any merge.

Comment (ElijahCaine) [inline etcd/os-faq.md]: I like to think of it as un-obfuscated.

File: etcd/os-faq.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+# How do I provision machines with etcd and config?
+
+The preferred way to provision *any* Container Linux machine is to write a [Container Linux Config][clconfig]. This config allows easy configuration of networking, storage, users, docker, flannel, etcd, systemd, update channel, image verification, and locksmith.
+
+As part of the installation process, this YAML file is converted into an Ignition config. On first system boot the Container Linux node is configured into the desired state specified in the Container Linux Config. See an example of how to use this in the [Getting started with etcd][getting-started] guide.
+
+# Can etcd be configured with cloud-config?
+
+The preferred way to configure a Container Linux machine is with Container Linux Configs and Ignition.
```
Comment (crawford) [inline etcd/os-faq.md]: Maybe a link to https://github.com/coreos/container-linux-config-transpiler/blob/master/doc/examples.md#etcd?

File: etcd/os-faq.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+The etcd2 binary will be included in the OS until June, 2018.
+
+The recommended way to run etcd on Container Linux is to use the etcd-member service.
+
+## When will v3 be included in the OS image?
+
+Version 3-series etcd binaries are packaged in a Linux container and instantiated by the `etcd-member.service` systemd unit.
+
+# Can I curl the v3 api?
```
Comment (xiang90) [inline etcd/os-faq.md]: this is not accurate. etcd3 ships with a http gateway for v3 api: see https://github.com/coreos/etcd/blob/master/Documentation/dev-guide/api_grpc_gateway.md

**Pull request merged**